### ISSUE TITLE
PyQt5 Hooks fix does not work PyQt 5.10.1

### ISSUE THREAD

**Issue by Devligue** [pyinstaller/pyinstaller#3474]
In the past I had some problems with pyinstaller and pyqt5 (#3338) which were solved by #3439 (and its precursor #3233). But only recently I moved from PyQt 5.9.1 to 5.10.1 and to my surprise I encounter the same problems as prior to #3439 (#3233).

I have tried installing various versions of pyinstaller and they fix my #3338 issue under PyQt 5.9.1, but not under PyQt 5.10.1. This is why I felt like letting you guys know about that problem comeback.

**Comment by Devligue** [comment]
@bjones1 sure. My particular problem is exactly the same as I described it in #3338. Long story short normally my application correctly uses `QWindowsVistaStyle` but when packaged with PyInstaller it uses the only-available `QWindowsStyle`.

When I had this issue on PyQt 5.9.1 your #3233 fork fixed it, and now that I upgraded to PyQt 5.10.1 the problem is once more. I have tried upgrading all my project dependencies but the problem persisted. It was only downgrading PyQt back to 5.9.1 that made it work again. Sadly I won't be able to test intermediate versions (5.9.2 and 5.10.0), until monday.

Test code is available below (will produce different output when packaged):
```
import sys
from PyQt5 import QtWidgets

if __name__ == '__main__':
    app = QtWidgets.QApplication(sys.argv)
    print('Currently used style:', app.style().metaObject().className())
    print('Available styles:', QtWidgets.QStyleFactory.keys())
```

**Comment by FranzPio** [comment]
@Devligue Have you installed the latest PyInstaller development version by
``pip install https://github.com/pyinstaller/pyinstaller/archive/develop.tar.gz``?
I'm on version 3.4.dev0+eea47968c and the styles now work for me.

**Comment by bjones1** [comment]
@Devligue, what version/branch of PyInstaller are you testing with? As @FranzPio says, I'd like to do any testing on the current development version, which has a few more improvements and tweaks compared to #3233.

**Comment by Devligue** [comment]
I was testing both with the #3233 and the develop branch (I did `pip install git+https://github.com/pyinstaller/pyinstaller@develop`) with similar effect. But I will test it again just to make sure.

**Comment by Devligue** [comment]
Ok @bjones1, I double checked and on PyQt 5.10.1 when I install PyInstaller with `pip install https://github.com/pyinstaller/pyinstaller/archive/develop.zip` the styles definitely won't work for me.

BTW @FranzPio, how do you find the exact version of installed package? When I do `pip list` it just shows me `PyInstaller 3.3.1` whether I install it by `pip install pyinstaller` or `pip install https://github.com/pyinstaller/pyinstaller/archive/develop.zip`, and you somehow got this `3.4.dev0+eea47968c`.

For the record I am actually using `pipenv`. I don't know if that changes anything.

**Comment by FranzPio** [comment]
@Devligue I used PyInstaller's ``--version`` flag to get that.

**Comment by htgoebel** [comment]
@Devligue If `pyinstaller --version` returns 3.3.1 you are obviously *not* using 3.4.dev0+b31794cf1. Please check your environment.

**Comment by bjones1** [comment]
I'm closing this for now, assuming it's an environment problem. Reopen if not.

**Comment by Devligue** [comment]
Sorry for keeping you waiting. After some additional testing it turned out that `pipenv` was at fault of overriding the develop branch version with the latest released 3.3.1. I reported it in [this issue](https://github.com/pypa/pipenv/issues/2158). 

With dependencies installed manually everything worked fine, ergo the `develop` branch solved my issues under PyQt 5.10.1. Thank you, and sorry for the problem!

**Comment by bjones1** [comment]
Thanks for the followup and confirmation that the Pyinstaller code works.
